winpr/libwinpr/CMakeLists.txt

```diff
@@ -87,6 +87,17 @@ if (TIMER_CREATE AND TIMER_DELETE AND TIMER_SETTIME AND TIMER_GETTIME)
 	winpr_library_add_private(rt)
 endif()
 
+set(CMAKE_REQUIRED_INCLUDES backtrace.h)
+check_function_exists(backtrace BACKTRACE)
+if (NOT BACKTRACE)
+    set(CMAKE_REQUIRED_LIBRARIES execinfo)
+    check_function_exists(backtrace EXECINFO)
+    if (NOT EXECINFO)
+        message(FATAL_ERROR "Could not find backtrace function")
```

Comment:
Pleas no fatal errors, linux is not the only platform we support.

Reply:
Fixed.